qa/workunits/rbd/concurrent.sh

```diff
@@ -88,9 +88,15 @@ function setup() {
 
 	# Set up some environment for normal teuthology test setup.
 	# This really should not be necessary but I found it was.
-	TOP="/tmp/cephtest"
+
+        if [ -f /etc/ceph/ceph.conf ]; then
+            TOP="/etc/ceph"
+            export CEPH_ARGS="${CEPH_ARGS} --keyring ${TOP}/ceph.client.0.keyring"
+        else
+            TOP="/tmp/cephtest"
+            export CEPH_ARGS="${CEPH_ARGS} --keyring ${TOP}/data/client.0.keyring"
+        fi
 	export CEPH_ARGS="--conf ${TOP}/ceph.conf"
-	export CEPH_ARGS="${CEPH_ARGS} --keyring ${TOP}/data/client.0.keyring"
```

Comment:
That said, /tmp/cephtest is now /home/cephtest and teuthology is supposed to take care of all of --conf and --keyring.